src/fsharp/Fsc/fsc.exe.config

```diff
@@ -5,13 +5,12 @@
     <gcServer enabled="true"/>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
-        <assemblyIdentity
-          name="FSharp.Core"
-          publicKeyToken="b03f5f7f11d50a3a"
-          culture="neutral"/>
-        <bindingRedirect
-          oldVersion="2.0.0.0-4.4.1.0"
-          newVersion="4.4.1.0"/>
+        <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
+        <bindingRedirect oldVersion="2.0.0.0-4.4.1.0" newVersion="4.4.1.0"/>
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="System.Collections.Immutable" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-1.2.1.0" newVersion="1.2.1.0" />
```

Comment:
@dsyme 

Yeah ... we are in actual dll hell now.  I have spent most of the last two days trying to get our build working again.  which included reverting back to System.Collections.Immutable 1.2.0.

Parts of vs require and deploys 1.2.0, roslyn is build using 1.2.1, VS has binding redirects back down to 1.2.0.

Anyway I think this will be taken care of by: https://github.com/Microsoft/visualfsharp/pull/2675 which was supposed to just add sourcelink support, but now kind of fixes our current dll hell issues.

As a side note:  when code reviewing we need to be especially careful when looking at changes to dependent packages.

I am actually a bit mystified why the CI is continuing to work, although I suppose it may be that it reuses images, and they have parts of previous package downloads.  
Our internal microbuild has failed nightly for the last 4 nights, and I definitely see failures on all of my machines.
I am considering adding a git clean -fxdq to our CI.

Kevin